week4/week4-080.Dice/src/Dice.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Random;
+
+public class Dice {
+    private Random random;
+    private int numberOfSides;
+
+    public Dice(int numberOfSides) {
+        this.numberOfSides = numberOfSides;
+        random = new Random();
```

Comment:
Budi konzistentan. Preporučljivo je kod konstruktora/getera/setera i uopšte pristupa atributima neke klase koristiti "this". Ako ne koristiš, nemoj nigdje (što nije "idiomatski" pristup u javi), a ako koristiš, koristi svuda:

`this.random = new Random();`